.sanity-ansible-ignore-2.10.txt

```diff
@@ -0,0 +1,3 @@
+plugins/modules/nbde_server_tang.py pylint:blacklisted-name
+plugins/modules/nbde_server_tang.py pylint:ansible-format-automatic-specification
+plugins/modules/nbde_server_tang.py validate-modules!skip
```

Comment:
We're using ansible 2.9 for ansible-test since that's what Automation Hub uses - https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/config_files/tox-default.ini#L248 - so we can have a  .sanity-ansible-ignore-2.10.txt for future use, but for now we need .sanity-ansible-ignore-2.9.txt

Reply:
Nbde_server already has .sanity-ansible-ignore-2.9.txt.
https://github.com/linux-system-roles/nbde_server/blob/master/.sanity-ansible-ignore-2.9.txt

I added .sanity-ansible-ignore-2.10.txt since I found you put it in kernel_settings. :) We'd better not have 2.10?

Reply:
It is ok to have 2.10